content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -29,137 +29,130 @@ further_reading:
+
+### Required environment variables
 
 {{< tabs >}}
 
 {{% tab "Windows" %}}
 
-If your application runs in IIS, skip the rest of this section.
-
-For Windows applications **not** running in IIS, set these two environment variables before starting your application to enable automatic instrumentation:
+#### Applications not hosted in IIS
 
-**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
+To enable automatic instrumentation on For Windows applications not in IIS, you must set two environment variables before starting your application:
 
 Name                       | Value
 ---------------------------|------
 `CORECLR_ENABLE_PROFILING` | `1`
 `CORECLR_PROFILER`         | `{846F5F1C-F9AE-4B07-969E-05C26BC060D8}`
 
-#### Windows Services
+<div class="alert alert-warning"> 
+  <strong>Note:</strong> The .NET runtime tries to load a profiler into any .NET process started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. Don't set these environment variables globally because this causes all .NET processes on the host to load the profiler.
+</div>
 
-To automatically instrument a Windows Service, set the environment variables for that Service in the Windows Registry. Create a multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\<SERVICE NAME>` key. Then set the key's data to the values in the table:
-```text
-CORECLR_ENABLE_PROFILING=1
-CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
-```
+#### Windows services
+
+To automatically instrument a Windows service, set the `CORECLR_ENABLE_PROFILING` and `CORECLR_PROFILER` environment variables: 
 
-This can be done either through the Registry Editor as in the image below, or through a PowerShell snippet:
+1. In the Windows Registry Editor, create a multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\<SERVICE NAME>` key. 
+2. Set the value data to:
+    ```text
+    CORECLR_ENABLE_PROFILING=1
+    CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+    ```
+    {{< img src="tracing/setup/dotnet/RegistryEditorCore.png" alt="Creating the environment variables for instrumenting a service in the Registry Editor" >}}
 
-{{< img src="tracing/setup/dotnet/RegistryEditorCore.png" alt="Registry Editor"  >}}
+Alternatively, you can set the environment variables by using the following PowerShell snippet:
 
 ```powershell
 [String[]] $v = @("CORECLR_ENABLE_PROFILING=1", "CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}")
 Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<NAME> -Name Environment -Value $v
 ```
 
-#### Console Apps
+#### Console applications
```

Comment:
"Console applications" falls under "Applications not hosted in IIS" above. Should it use `#####`?
```suggestion
##### Console applications
```